build.sbt

```diff
@@ -79,5 +102,11 @@ lazy val scalaz = (project in file("scalaz"))
   .settings(moduleName := "github4s-scalaz")
   .settings(buildSettings: _*)
   .settings(scalazDependencies: _*)
-  .dependsOn(github4s)
+  .dependsOn(github4sJVM)
   .enablePlugins(AutomateHeaderPlugin)
+
+lazy val testSettings = Seq(
+  fork in Test := false
+)
+
+addCommandAlias("testAllCoverable", ";docs/test;scalaz/test;github4sJVM/test")
```

Comment:
OK, I switched to using a higher version of the sbt-scoverage plugin and proved to be compatible with ScalaJS, so I removed the extra command and all the tests go through the coverage process.